include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -60,10 +65,22 @@ class Indentation {
     /// @param z Indentation leven on the z-axis
     explicit Indentation(std::uint8_t x, std::uint8_t y, std::uint8_t z);
 
+    Indentation(const Indentation& indentation);
```

Comment:
Make sure you run clang-format before you merge.

Reply:
This should be `explicit`.

Reply:
It's a copy constructor. You sure :D?